spec/repo_spec.rb

```diff
@@ -1,10 +1,25 @@
 describe CuttingEdge::Repository do
+  
+  it 'is capable of being hidden' do
+    expect(CuttingEdge::Repository.new('org', 'name').hidden?).to be false
+    expect(CuttingEdge::Repository.new('org', 'name', nil, nil, nil, nil, nil, true).hidden?).to be true
```

Comment:
Wow, this method call; all to set the repo to hidden? 😕

Reply:
Yes it's ugly. Alternatively, we could work with named parameters/a Hash here?

Reply:
Yes, named parameters would be nice!